docs/examples/http_crawler.ts

```diff
@@ -48,6 +48,8 @@ const crawler = new HttpCrawler({
 
 // Run the crawler and wait for it to finish.
 // It will crawl a list of URLs from an external file, load each URL using a plain HTTP request, and save HTML
-await crawler.run(['https://crawlee.dev']);
+await crawler.run([
+    'https://crawlee.dev',
+]);
```

Comment:
update your fork and remove those changes, we now have a formatter integrated, so you can just run `yarn format` to fix these things.

Reply:
its a clone not fork this time :)